### ISSUE TITLE
ComposeNamingLowercase false positive with Circuit

### ISSUE THREAD

**Issue by eboudrant** [slackhq/compose-lints#168]
Do you think the `ComposeNamingLowercase` lint rule (and others?) should be Circuit aware, since `"ComposeNamingLowercase"` don't apply to Circuit's presenters which return a state: https://slackhq.github.io/circuit/presenter/

```
@Composable
fun CounterPresenter(): CounterState {
  var count by rememberRetained { mutableStateOf(0) }
  ...
```

**Comment by ZacSweers** [comment]
We expose a config option for this case and recommend just using that. Does that work?

```xml
<issue id="ComposeNamingUppercase,ComposeNamingLowercase">
    <option name="allowed-composable-function-names" value=".*Presenter"/>
</issue>
```

**Comment by eboudrant** [comment]
Nice, I actually use 
```
    <issue id="ComposeNamingLowercase" >
        <!-- Exclude Circuit's presenters, don't apply (https://slackhq.github.io/circuit/presenter/) -->
        <ignore regexp="/presenter/" />
    </issue>
```
But I'll use your suggestion. Closing issue.
